src/components/SearchResultList/SearchResultList.tsx

```diff
@@ -0,0 +1,16 @@
+// FIXME(cheolee): resolve eslint errors
+/* eslint-disable react/prop-types */
+import { SearchItem } from '../../models/SearchItem';
+import { SearchResultRow } from './SearchResultRow';
+
+type Props = {
+  searchItems: SearchItem[];
+};
+
+export const SearchResultList: React.FC<Props> = ({ searchItems }) => (
```

Comment:
```suggestion
export const SearchResultList = ({ searchItems }: Props) => (
```

혹시 FC 대신 바로 프롭스에 타입을 지정해 주는건 어떨까요?
저게 평상시에는 괜찮은데, 혹시라도 디폴트 프롭스를 썼을때 문제가 생길 수가 있습니다. ㅠ
칠드런의 경우도 타입을 차라리 명확히 지정해 주는게 더 좋다고 생각하구요!

Reply:
1. 디폴트 프롭스를 썼을 때 어떤 문제가 발생하나요?
2. React.FC<P>를 사용하면, react.d.ts에서 정의된 children의 type으로 타이핑 되기 때문에 직접 타이핑하는 것보다 더욱 안전하다 생각이 듭니다.

Reply:
https://codesandbox.io/s/xenodochial-rhodes-17z8j?file=/src/Hello.tsx

Reply:
![image](https://user-images.githubusercontent.com/36758131/79062563-66565180-7cd6-11ea-9c26-9ce4b2ccc8d0.png)